exchanges/retry/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
```

Comment:
This is cool, but does our bundling process currently support aliasing?

I stopped using aliases after getting tired of writing alias maps in multiple places (tsconfig and webpack config)

Reply:
Everything seems fine as far as I can see - happy to swap them to be relative paths if preferred though